frontend/tests/utils/authHelpers.ts

```diff
@@ -37,6 +39,49 @@ export async function registerUser(page: Page, name: string, email: string, pass
   return userId;
 }
 
+// Verify email address; returns a Page promise that should be used to load a UserDashboardPage or AdminDashboardPage
+// Does not call loginAs(page.request, tempUser.email, tempUser.password); calling code should be doing that
+export async function verifyTempUserEmail(page: Page, tempUser: TempUser): Promise<Page> {
+  const emailPage = await tempUser.mailbox.openEmail(page, EmailSubjects.VerifyEmail);
+  const pagePromise = emailPage.page.context().waitForEvent('page');
+  await emailPage.clickVerifyEmail();
+  return pagePromise;
+}
+
+export async function addUserToOrg(api: APIRequestContext, userId: string, orgId: string, role: string): Promise<unknown> {
```

Comment:
I don't love doing the role this way, but I'm fine with it as a workaround. But could we restrict the role to valid values? eg: `'MANAGER' | 'EDITOR'`?

Reply:
Done in commit 531bd7c5.